### PR TITLE
Update API changes for 2019.2

### PR DESCRIPTION
**Pull request by pvanderknyff**
Update documentation for new javascript helper methods in 2019.2. Does not include company and support link documentation, still working on those.

### CODE REVIEW

File: docs/api-reference.md [hunk 1]
```diff
@@ -152,12 +153,24 @@ Format the attributes as 'key=value'. By default, some values are escaped or wra
...
+
 Example:
 
     formattedParams.push(connectionHelper.FormatKeyValuePair(key, params[key]));
 
     params[connectionHelper.keywordODBCUsername] = attr[connectionHelper.attributeUsername];
 
+_Throw Tableau Exception_
+
+Normally, throwing an exception in in a javascript componant will show the user a more generic error message in the product. To have a custom error message appear in Tableau, use the following format:
```
Comment (lukewrites) [inline docs/api-reference.md]: Spelling nit: `component`

File: docs/api-reference.md [hunk 2]
```diff
@@ -152,12 +153,24 @@ Format the attributes as 'key=value'. By default, some values are escaped or wra
...
+
 Example:
 
     formattedParams.push(connectionHelper.FormatKeyValuePair(key, params[key]));
 
     params[connectionHelper.keywordODBCUsername] = attr[connectionHelper.attributeUsername];
 
+_Throw Tableau Exception_
+
+Normally, throwing an exception in in a javascript component will show the user a more generic error message in the product. To have a custom error message appear in Tableau, use the following format:
```
Comment (kpwilson) [inline docs/api-reference.md]: Remove duplicate "in"

**Comment by kpwilson** [comment]
I found a duplicate "in" in the text. Otherwise, after a quick copyedit, the text looks good.

File: docs/api-reference.md (hunk 2 above)
Comment (lukewrites) [reply docs/api-reference.md]:
```suggestion
Normally, throwing an exception in a JavaScript component will show the user a more generic error message in the product. To have a custom error message appear in Tableau, use the following format:
```

**Pull request merged**